pkg/jobs/registry_test.go

```diff
@@ -688,6 +689,12 @@ func TestJitterCalculation(t *testing.T) {
 		return time.Duration(float64(input) * minFactor), time.Duration(float64(input) * maxFactor)
 	}
 
+	// Chosen by a fair dice roll. We use a determinstic seed to avoid the case
+	// where the test flakes because the random value rounds to 1/2. This is more
+	// common than you would expect because its probability is based on the width
+	// of the duration, not the width of the random float.
+	rand.Seed(4)
```

Comment:
I was a little concerned that setting this global seed might impact other randomized tests. But then I read that perhaps this isn't doing what we want in the first place:

```
// As of Go 1.24 [Seed] is a no-op. To restore the previous behavior set
// GODEBUG=randseednop=0.
```

Interestingly, I think that comment is wrong. You need to set randseednop=1

Reply:
Good point on both counts. I switched strategies and rewrote the test to retry cases where input == output.